### ISSUE TITLE
Add HTTPS to cloudamqp_security_firewall

### ISSUE THREAD

**Issue by ArieLevs** [cloudamqp/terraform-provider-cloudamqp#95]
currently [cloudamqp_security_firewall](https://registry.terraform.io/providers/cloudamqp/cloudamqp/latest/docs/resources/security_firewall) does not support HTTPS service as [defined here](https://github.com/cloudamqp/terraform-provider-cloudamqp/blob/master/cloudamqp/resource_cloudamqp_security_firewall.go#L33)

please note that the [official documentation](https://www.cloudamqp.com/docs/product_overview.html#firewall) state that 
```
CloudAMQP is using port 22 and 443 for monitoring and statistics purposes, therefore they will always remain open.
```
although if you login to the console you can see it was added to CloudAMQP, so its currently impossible to use terraform without manually touching the console.

**Comment by dentarg** [comment]
@ArieLevs Thanks for raising this issue, we will adress it shortly

**Comment by dentarg** [comment]
> the [official documentation](https://www.cloudamqp.com/docs/product_overview.html#firewall) state that
> 
> ```
> CloudAMQP is using port 22 and 443 for monitoring and statistics purposes, therefore they will always remain open.
> ```

This has now been corrected

**Comment by dentarg** [comment]
I merged #96 with a fix for this, hopefully we can cut a new release with it tomorrow

**Comment by dentarg** [comment]
Released in 1.9.2: https://github.com/cloudamqp/terraform-provider-cloudamqp/releases/tag/v1.9.2

**Comment by ArieLevs** [comment]
@dentarg thanks for the quick response!
